_translator-files/po/fr/Installation-for-Macintosh.po

```diff
@@ -55,34 +55,33 @@ msgstr "# Installation sous macOS"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Macintosh.md:13
 msgid "Make sure you've already read the [Getting Started](Getting-Started) page."
-msgstr "Assurez-vous d'avoir déjà lu la page [Configuration](Getting-Started)."
+msgstr "Assurez-vous d'avoir déjà lu la page [Démarrage](Getting-Started)."
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Macintosh.md:15
 msgid "Upgrading? You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first."
-msgstr "Mise à jour ? Vous pouvez d'abord [sauvegarder votre configuration](Software-Manual#sauvegarde-de-jamulus)."
+msgstr "Mise à jour ? Vous pourriez d'abord souhaiter [sauvegarder votre configuration](Software-Manual#sauvegarde-de-jamulus)."
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Macintosh.md:17
-#, fuzzy
-#| msgid "[Download Jamulus (Intel)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}"
 msgid "[Download Jamulus (Universal build)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}\\\\"
-msgstr "[Téléchargez Jamulus (Intel)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}"
+msgstr "[Téléchargez Jamulus (construction universelle)]({{ site.download_root_link }}{{ site.download_file_names.mac }}){: .button}\\\\"
```

Comment:
_translator-files/po/de/Installation-for-Macintosh.po line 68 above has this untranslated.  Should they be consistent?

Reply:
@trebmuh ?

Reply:
I think the german translation is incorrect.